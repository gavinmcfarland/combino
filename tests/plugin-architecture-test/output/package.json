{
	"name": "testproject",
	"version": "1.0.0",
	"description": "Test package for plugin architecture",
	"author": "Test Author",
	"scripts": {
		"test": "echo \"Test script\""
	},
	"keywords": ["test", "plugin", "architecture"],
	"pluginProcessed": true
}
