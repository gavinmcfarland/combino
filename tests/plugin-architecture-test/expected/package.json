{
	"name": "<%= name.toLowerCase() %>",
	"version": "<%= version %>",
	"description": "Test package for plugin architecture",
	"author": "<%= author %>",
	"scripts": {
		"test": "echo \"Test script\""
	},
	"keywords": [
		"test",
		"plugin",
		"architecture"
	],
	"pluginProcessed": true
}
